communications/slack.md

```diff
@@ -0,0 +1,91 @@
+## On Slack
```

Comment:
we should probably try to organize the techniques laid out here under the following categories (order subject to discussion)

> ## On Slack
> 
> abstract/high-level gist
> 
> ### Channel
> 
> use of star, section, conversation display mode, etc.
> 
> ### Thread
> 
> Official thread features, such as subscribe (which is automatically done through conversation participation), unsubscribe. And our additionally developed conventions to further utilize it.
> 
> Articulate the advantage of it, capture some screenshot of channels with and without regular thread usage, specifically that detailed thread conversations would not constantly trigger notifications to channel members, and the number of conversations exchanged can be seen as an indicator of "significance" (or lower count often represents a "smooth" operation without much discussion needed)
> 
> ### Notification
> 
> Global notification control (including do not disturb mode), per-workspace and per-channel notification control
>
> ### Remind
> 
> About how to leverage remind features (in actions - remind me about this) or `/remind` slash command
>
> ### Search
> 
> Highlight some more practical examples with the use of different combinations of `in:`, `from:`, `to:`, and various in-UI filters (such as date range)
> 
> ### DM
> 
> Articulate on when to use DM (such as non-work related, or credentials only), this applies to adhoc DM group messages. Emphasize how the information is no archived and searchable, and provides less transparency.